GeometryCore/src/Mesh.cpp

```diff
@@ -208,4 +210,55 @@ namespace geom_utils
 		}
 		return foundation;
 	}
+	Mesh integrateFoundationIntoModel(const Mesh& model, Mesh& foundation)
```

Comment:
`Mesh& foundation`
`Mesh foundation`
Tell me please whats the difference between those two variants of declarations?

Reply:
If indicated & this means that foundation address in memory,and which refers to the type Mesh. If without & this mean that we will use an initialized mesh object

Reply:
As far as I understand I need to use the second variant?

Reply:
Although I do not fully understand why

Reply:
& - means reference to the object, and if you change `foundation` object inside function, you actually change the original object passed to that function. Do you make such side-effect for the foundation argument intentionally? If yes, please document that somewhere. Otherwise user of that function will have a kind of surprise)
On the other hand, passing argument by value will create copy of the object.